src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -238,12 +256,12 @@ uiModules.get('apps/management')
       : undefined;
 
     // this seems wrong, but it's the original logic... https://git.io/vHYFo
-    const notExpandable = (!expandable && this.canExpandIndices())
+    const notExpandable = (this.canExpandIndices() && !expandable)
       ? true
       : undefined;
 
     // Only event-time-based index patterns set an intervalName.
-    const intervalName = (nameIsPattern && nameInterval)
+    const intervalName = (this.canUseTimePattern() && nameIsPattern && nameInterval)
```

Comment:
Since the select field that sets the `nameInterval` is now hidden behind a more aggressive `ng-if`, do you think we only need to check if `nameInterval` exists at this point? It feels the first two conditions are redundant